maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java

```diff
@@ -184,32 +184,32 @@ public void validateRawModel( Model m, ModelBuildingRequest request, ModelProble
                 }
 
                 validate30RawProfileActivation( problems, profile.getActivation(), profile.getId(),
-                                                prefix + ".activation", request );
+                                                prefix, ".activation", request );
```

Comment:
I wonder whether we should not drop the leading dot here and add it in the method only. This makes it consistent with the rest which does not use leading dots for premature concat. Or actually the prefix should contain the dot because it is a *prefix*.